docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -93,6 +97,36 @@ To add a tag, follow these steps:
 1. Select a value from the **Select value** drop-down. This field is populated based upon the tag.
 1. Optional: Select **+** to add an additional tag.
 
+## Optional: Use Aggregate by
+
+{{% admonition type="warning" %}}
+**Aggregate by** is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.
+[Enable the `metricsSummary` feature toggle](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
+{{% /admonition %}}
+
+Using **Aggregate by**, you can calculate RED metrics (span count, erroring span count, and latency information) for `kind=server` spans recently sent to Tempo, grouped by one or more attributes.
+
+Whatever is selected in the **Aggregate by** list, you'll see every combination of present in your data.
+Each aggregate value, for example `intrinsic`:`name`, has a corresponding column in the results table.
+The RED metrics are calculated for every endpoint and user agent combinations found in your data.
```

Comment:
I don't understand this right now, but maybe it will make more sense once I see the screenshot.

Reply:
Screenshot: 
![image](https://github.com/grafana/grafana/assets/104772500/5380901b-b57d-4946-8d95-2e365bd9beaf)

Reply:
Got it. In this case, my suggestion would be 

`The RED metrics are calculated for every name and user agent combination found in your data.`

Suggest using `name` instead of `endpoint` since that's the exact thing shown in the selector.

Reply:
```suggestion
The RED metrics are calculated for every name and user agent combination found in your data.
```